js/utils/constants.js

```diff
@@ -859,9 +1133,151 @@
+         * @constant OO.VIDEO.ENCODING.WIDEVINE_HLS
+         * @type {string}
+         */
+        WIDEVINE_HLS: "wv_hls",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.WIDEVINE_WVM ('wv_wvm')</code>:
+         *   An encoding type for widevine wvm streams.
+         * @constant OO.VIDEO.ENCODING.WIDEVINE_WVM
+         * @type {string}
+         */
+        WIDEVINE_WVM: "wv_wvm",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.WIDEVINE_MP4 ('wv_mp4')</code>:
+         *   An encoding type for widevine mp4 streams.
+         * @constant OO.VIDEO.ENCODING.WIDEVINE_MP4
+         * @type {string}
+         */
+        WIDEVINE_MP4: "wv_mp4",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.RTMP ('rtmp')</code>:
+         *   An encoding type for non-drm rtmp streams.
+         * @constant OO.VIDEO.ENCODING.RTMP
+         * @type {string}
+         */
+        RTMP: "rtmp",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.AUDIO ('audio')</code>:
+         *   An encoding type for non-drm audio streams.
+         * @constant OO.VIDEO.ENCODING.AUDIO
+         * @type {string}
+         */
+        AUDIO: "audio",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.SMOOTH ('smooth')</code>:
+         *   An encoding type for non-drm smooth streams.
+         * @constant OO.VIDEO.ENCODING.SMOOTH
+         * @type {string}
+         */
+        SMOOTH: "smooth",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.FAXS_HLS ('faxs_hls')</code>:
+         *   An encoding type for adobe faxs streams.
+         * @constant OO.VIDEO.ENCODING.FAXS_HLS
+         * @type {string}
+         */
+        FAXS_HLS: "faxs_hls",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.AKAMAI_HD_VOD ('akamai_hd_vod')</code>:
+         *   An encoding type for akamai hd vod streams.
+         * @constant OO.VIDEO.ENCODING.AKAMAI_HD_VOD
+         * @type {string}
+         */
+        AKAMAI_HD_VOD: "akamai_hd_vod",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.AKAMAI_HD2_VOD_HLS ('akamai_hd2_vod_hls')</code>:
+         *   An encoding type for akamai hd2 vod hls streams.
+         * @constant OO.VIDEO.ENCODING.AKAMAI_HD2_VOD_HLS
+         * @type {string}
+         */
+        AKAMAI_HD2_VOD_HLS: "akamai_hd2_vod_hls",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.AKAMAI_HD2_VOD_HDS ('akamai_hd2_vod_hds')</code>:
+         *   An encoding type for akamai hd2 vod hds streams.
+         * @constant OO.VIDEO.ENCODING.AKAMAI_HD2_VOD_HDS
+         * @type {string}
+         */
+        AKAMAI_HD2_VOD_HDS: "akamai_hd2_vod_hds",
+
+        /**
+         * @description <code>OO.VIDEO.ENCODING.IMA ('ima')</code>:
+         *   A string that repsents a video stream that is controlled and configured directly by IMA.
+         * @constant OO.VIDEO.ENCODING.IMA
+         * @type {string}
+         */
+        IMA: "ima"
+      }
+
+    };
+
```

Comment:
That's a good point, I can change it.  The reason it is the way it is is because I listed them in rough order of popularity and grouped them by encryption type.  All clear encodings appear first, Widevine is next, "assorted" are next, akamai is listed last, and the IMA one is not technically a true encryption type but is used to differentiate streams played by the IMA VTC plugin